user.py

```diff
@@ -16,7 +16,7 @@
 any other part of the program, but is included in order to accomodate
 their inclusion in the future (see https://github.com/lineaba/prog-o-meter/issues/33).
 """
-
+from database import DataBaser as DBser
```

Comment:
If you change the name of the database object, remember to also change it here :)